app/src/main/res/values/strings.xml

```diff
@@ -127,19 +127,14 @@
     </string>
     <string name="about_background_heading">Background</string>
     <string name="about_background_body_text">
-        We\'ve all been there: It's one of those group situations, where, after a little while, the
-        group branches off into pairs. Jenny, the runner, goes
-        with Fred, also a runner. James always watches indie flicks. He\'ll end up with April
-        because she does the same.\n
+        We've all been there: messages that go nowhere; likes, winks, and pokes sent with no response; 
```

Comment:
The content of frustration of interaction with low quality matches is a good start to segue into high mutual attraction. You expressed it with good conversation tone ("We've all been there") and added sentences phrases that capture the online experience (e.g. "messages that go nowhere").

Nitpicking, the content of the list could be improved. Having a semi-colon list with comma sub lists is good for an article or longer writing. For the screen writing, it seems a bit too complicated to catch the attention. Also, the particular list items could be stronger. Nowadays, many people know that pokes and winks usually go nowhere. Likes are often only visible to premium subscribers of the online app. That makes it unsurprising to the avid online dater. Yet, in the area of online dating, you have a lot of other choices that actually happen and resonate with people.

Creepy messages are annoying. Getting a million "Hi" and "how are you messages is annoying. Messaging forever without being asked out is annoying. Spending hours after hours swiping and getting nowhere is a pain.